src/stylesheets/abstracts/_media-queries.scss

```diff
@@ -0,0 +1,105 @@
+//
+//  MEDIA QUERIES
+//––––––––––––––––––––––––––––––––––––––––––––––––––
+//  From: https://glennmccomb.com/articles/useful-sass-scss-media-query-mixins-for-bootstrap/
+
+// A map of breakpoints.
+$breakpoints: (
+  xs: 576px,
+  sm: 768px,
+  md: 992px,
+  lg: 1200px
+);
+
+
+//
+//  RESPOND ABOVE
+//––––––––––––––––––––––––––––––––––––––––––––––––––
+
+// @include respond-above(sm) {}
+@mixin respond-above($breakpoint) {
+
+  // If the breakpoint exists in the map.
+  @if map-has-key($breakpoints, $breakpoint) {
+
+    // Get the breakpoint value.
+    $breakpoint-value: map-get($breakpoints, $breakpoint);
+
+    // Write the media query.
+    @media (min-width: $breakpoint-value) {
+      @content;
+    }
+  
+  // If the breakpoint doesn't exist in the map.
+  } @else {
+
+    // Log a warning.
+    @warn 'Invalid breakpoint: #{$breakpoint}.';
+  }
+}
+
```

Comment:
$mq-phone: 400px;
$mq-phone-wide: 480px;
$mq-phablet: 560px;
$mq-tablet-small: 640px;
$mq-tablet: 768px;
$mq-tablet-wide: 1024px;
$mq-desktop: 1248px;
$mq-desktop-wide: 1440px;

$breakpoints: (
        "phone":        $mq-phone,
        "phone-wide":   $mq-phone-wide,
        "phablet":      $mq-phablet,
        "tablet-small": $mq-tablet-small,
        "tablet":       $mq-tablet,
        "tablet-wide":  $mq-tablet-wide,
        "desktop":      $mq-desktop,
        "desktop-wide": $mq-desktop-wide
);

@mixin mq($width, $type) {
    @if map_has_key($breakpoints, $width) {
      $width: map_get($breakpoints, $width);
      @if $type == max {
        $width: $width - 1px;
      }
      @media only screen and (#{$type}-width: $width) {
        @content;
      }
    }
  }

.my_elem{
color:blue;
@include mq(mobile){
    color:red;
 }
}
@viktooar  check ,this light version media-query

}